python-qa-tools/stats/stats.py

```diff
@@ -0,0 +1,119 @@
+import json, logging
+from db import db
+
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+COL_NAME = "test_stats"
+TX_TYPE = "tx"
+QUERY_TYPE = "query"
+
```

Comment:
```suggestion
```

Reply:
This is due to python formatter. Every function in python will be separated from other with two empty lines.